src/backer/backering.py

```diff
@@ -407,6 +411,8 @@ def __next__(self):
 
                     if self.ledger and topic == "/receipt":
                         try:
+                            self.queue,push_to_queued(self.pre, msg)
```

Comment:
There is a `,` after `self.queue` here

Reply:
updated
ref here: 31d4b80f5e0b852bcb92202843308b587bc74f38